12_path_finding.asciidoc

```diff
@@ -351,7 +351,7 @@ Therefore, Selena's node must consider how long to keep this knowledge before as
 [[mpp]]
 === Multi-Path Payments (MPP)
 
-_Multi-Path Payments (MPP)_ are a feature that was introduced in the Lightning Network in 2020 and is already very widely available. Multi-Path Payments allow a payment to be split into multiple _parts_ which are sent as HTLCs over several different paths to the intended recipient, preserving the _atomicity_ of the overall payment. In this context, atomicity means that either all the HTLC parts of a payment are eventually fulfilled or the  entire payment fails and all the HTLC parts fail. There is no possibility of a partially successful payment.
+_Multi-Path Payments (MPP)_ is a feature that was introduced in the Lightning Network in 2020 and is already very widely available. Multi-Path Payments allow a payment to be split into multiple _parts_ which are sent as HTLCs over several different paths to the intended recipient, preserving the _atomicity_ of the overall payment. In this context, atomicity means that either all the HTLC parts of a payment are eventually fulfilled or the  entire payment fails and all the HTLC parts fail. There is no possibility of a partially successful payment.
```

Comment:
Hmm, yeh somewhat ambgiuous...I think it's correct as is since the P (payments) in MPP is plural